tests/spec/core/data-cite-spec.js

```diff
@@ -263,6 +263,67 @@ describe("Core — data-cite attribute", () => {
     });
   });
 
+  it("Adds title to a reference when inline-link is empty normative reference", async () => {
+    const ops = {
+      config: makeBasicConfig(),
+      body: `${makeDefaultBody()}
+        <section>
+          <p id="t1"><a data-cite="HTML"></a></p>
+          <p id="t2"><a data-cite="Fetch"></a></p>
+          <p id="t3"><a data-cite="HTML">This should not be replaced</a></p>
+        </section>
+      `,
+    };
+    const doc = await makeRSDoc(ops);
+    let a = doc.querySelector("#t1 > a");
+    expect(a.textContent).toBe("HTML Standard");
+    expect(a.href).toBe("https://html.spec.whatwg.org/multipage/");
+    expect(doc.getElementById("bib-html").closest("section").id).toBe(
+      "normative-references"
+    );
+    a = doc.querySelector("#t2 > a");
+    expect(a.textContent).toBe("Fetch Standard");
+    expect(a.href).toBe("https://fetch.spec.whatwg.org/");
+    expect(doc.getElementById("bib-fetch").closest("section").id).toBe(
+      "normative-references"
+    );
+    a = doc.querySelector("#t3 > a");
+    expect(a.textContent).toBe("This should not be replaced");
+    expect(a.href).toBe("https://html.spec.whatwg.org/multipage/");
+    expect(doc.getElementById("bib-fetch").closest("section").id).toBe(
+      "normative-references"
+    );
+  });
+
+  it("Adds title to a reference when inline-link is empty normative reference in definition", async () => {
+    const ops = {
+      config: makeBasicConfig(),
+      body: `${makeDefaultBody()}
+        <section>
+          <p id="t1"><dfn data-cite="WHATWG-HTML#test"></dfn></p>
```

Comment:
@marcoscaceres Is this also expected? Better than just being empty though...

Reply:
Yeah, that’s definitely valid to test for.